### ISSUE TITLE
Question about private Docker registry support

### ISSUE THREAD

**Issue by jwlai** [alien4cloud/alien4cloud#141]
Is there any support for images from a private repository? (Particularly with K8s plugin support). The examples are only public images.

**Comment by EdgarLGB** [comment]
Yes, you could modify ([for example here](https://github.com/alien4cloud/docker-tosca-types/blob/3bd300a9650cfc6c84cdf46e6ed5e15374ff5f93/containers-types/nodecellar-sample-types/nodecellar-sample-types.yml#L12)) the repository defined in your csar.
